lib/archethic/bootstrap.ex

```diff
@@ -207,6 +208,7 @@ defmodule Archethic.Bootstrap do
       Logger.info("Synchronization finished")
     end
 
+    Archethic.Bootstrap.NetworkConstraints.persist_genesis_address()
```

Comment:
You are already doing it at the end of the self-repair so why would you want to do it now ?

Reply:
ok